joml-geometry/src/main/java/org/terasology/joml/geom/Rectangled.java

```diff
@@ -391,51 +381,45 @@ public Vector2d lengths(Vector2d dest) {
     /**
      * Check if this rectangle contains the given <code>rectangle</code>.
      *
-     * @param rectangle
-     *          the rectangle to test
+     * @param rectangle the rectangle to test
      * @return <code>true</code> iff this rectangle contains the rectangle; <code>false</code> otherwise
      */
     @Override
     public boolean containsRectangle(Rectangledc rectangle) {
         return rectangle.minX() >= minX && rectangle.maxX() <= maxX &&
-               rectangle.minY() >= minY && rectangle.maxY() <= maxY;
+                rectangle.minY() >= minY && rectangle.maxY() <= maxY;
     }
 
     /**
      * Check if this rectangle contains the given <code>rectangle</code>.
      *
-     * @param rectangle
-     *          the rectangle to test
+     * @param rectangle the rectangle to test
      * @return <code>true</code> iff this rectangle contains the rectangle; <code>false</code> otherwise
      */
     @Override
     public boolean containsRectangle(Rectanglefc rectangle) {
         return rectangle.minX() >= minX && rectangle.maxX() <= maxX &&
-               rectangle.minY() >= minY && rectangle.maxY() <= maxY;
+                rectangle.minY() >= minY && rectangle.maxY() <= maxY;
     }
 
 
-
     /**
      * Check if this rectangle contains the given <code>rectangle</code>.
      *
-     * @param rectangle
-     *          the rectangle to test
+     * @param rectangle the rectangle to test
      * @return <code>true</code> iff this rectangle contains the rectangle; <code>false</code> otherwise
      */
     @Override
     public boolean containsRectangle(Rectangleic rectangle) {
         return rectangle.minX() >= minX && rectangle.maxX() <= maxX &&
-               rectangle.minY() >= minY && rectangle.maxY() <= maxY;
+                rectangle.minY() >= minY && rectangle.maxY() <= maxY;
```

Comment:
That's what the IntelliJ auto-formatter does :shrug: Since we have a common configuration committed in all our repos using that should be fine, or we should put some gradlew plugin to ensure consistent formatting.

I did not change anything by hand here.